docs/book/usage.md

```diff
@@ -51,24 +55,30 @@ var_dump($aggregator->getMergedConfig());
 The result of the above will be:
 
 ```php
-array(3) {
+array(5) {
   'parameter_usage' =>
   string(3) "bar"
   'parameter_name' =>
   string(5) "%foo%"
   'recursive_parameter_usage' =>
   string(3) "qoo"
+  'parameterized_parameter_usage' =>
+  string(3) "qoo"
```

Comment:
Same note as above; I'm not sure how it gets to `qoo` here, and, if it _is_ correct, I think there needs to be an explanation of _why_ in the documentation narrative.